google/cloud/forseti/services/scanner/dao.py

```diff
@@ -40,14 +40,17 @@ def define_violation(model_name, dbengine):
     """
 
     base = declarative_base()
-    violations_tablename = '{}_violations'.format(model_name)
+    # pylint: disable=too-many-format-args
+    violations_tablename = 'violations'.format(model_name)
+    # pylint: enable=too-many-format-args
 
     class Violation(base):
         """Row entry for a violation."""
 
         __tablename__ = violations_tablename
 
         id = Column(Integer, primary_key=True)
+        model_handle = Column(String(256))
         resource_type = Column(String(256), nullable=False)
```

Comment:
Please add resource_id back into the model, this got dropped. Also I agree that adding resource_full_name is a good idea.

Reply:
resource_id is added back; also taking the chance to fix how resource_id is being set that will make it more easy to identify which resource is the violation coming from.

I also started to implement the full_name, but it grew to be a sweeping change.  Let me do this in a follow-up PR, as I need to re-think how to make the easiest change.